components/style/input.css

```diff
@@ -0,0 +1,55 @@
+.input-group input{
+  display: block;
+  padding: 0.375rem 0.75rem;
+  font-size: 1rem;
+  line-height: 1.5;
+  color: #495057;
+  background-color: #fff;
+  border: 1px solid #ced4da;
+  border-radius: 0.25rem;
+  transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
+  margin: 1rem 0;
+}
+
+.input-group input:focus {
+  color: #495057;
+  background-color: #fff;
+  border-color: #80bdff;
+  outline: 0;
+  box-shadow: 0 0 0 0.2rem rgb(0 123 255 / 25%);
+}
+
+.input-group .input-lg {
+  padding: 0.5rem 1rem;
+  font-size: 1.25rem;
+  line-height: 1.5;
+  border-radius: .3rem;
+}
+
+.input-group .input-sm {
+  padding: 0.25rem 0.5rem;
+  font-size: .875rem;
+  line-height: 1.5;
+  border-radius: 0.2rem;
+}
+
+.input-group .input-disabled {
+  background-color: #e9ecef;
+  opacity: 1;
+}
+
+.invalid-input {
+  border-color: #dc3545 !important;
+}
+
+.input-group .invalid-input:focus {
+  border-color: #dc3545;
+  box-shadow: 0 0 0 0.2rem rgb(220 53 69 / 25%);
```

Comment:
Change:- Use var for colour.

Reply:
I'd used different colors in different branches, thus will convert them into variables in the final merge.